src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -506,17 +512,6 @@ export const MousePopulationsModel = types
         get graphButtons(): ToolbarButton[] {
           const buttons = [];
 
-          buttons.push({
-            title: "Scale",
-            type: "float-button",
-            value: self.showMaxPoints,
-            action: (val: boolean) => {
-              self.toggleShowMaxPoints();
-            },
-            floatCorner: "upper-right",
-            section: "data"
-          });
-
```

Comment:
Glad to see this removed!  I was never happy with how I implemented this as a `float-button`.